lint/src/main/kotlin/net/grandcentrix/thirtyinch/Issues.kt

```diff
@@ -0,0 +1,31 @@
+package net.grandcentrix.thirtyinch
+
+import com.android.tools.lint.detector.api.*
+import java.util.*
+
+enum class Issues(
```

Comment:
I don't like enum. Especially if we can just create a`data class` instead.
Addional, I don't see the reason why it is a enum. We cal also creta a "normal" class,,